app/models/ontology/validations.rb

```diff
@@ -0,0 +1,16 @@
+module Ontology::Validations
+  extend ActiveSupport::Concern
+
+  included do
+    validates_uniqueness_of :iri, if: :iri_changed?
+    validates_format_of :iri, with: URI::regexp(Settings.allowed_iri_schemes)
```

Comment:
Style/ColonMethodCall: Do not use `::` for method calls.<br>Rails/Validation: Use the new "sexy" validations (`validates` ...).